### ISSUE TITLE
Game Over Screen Crashes IKEMEN. + No Continue Music playing.

### ISSUE THREAD

**Issue by cyliamargatroid** [Windblade-GR01/Ikemen_GO#47]
```
./script/select.lua:2588: 3番目の引数が数ではありません。
stack traceback:
	[G]: in function 'playBGM'
	./script/select.lua:2588: in function 'f_continue'
	./script/select.lua:881: in function 'f_selectAdvance'
	script/main.lua:1610: in function 'f_mainMenu'
	script/main.lua:2205: in main chunk
	[G]: ?

**Comment by cyliamargatroid** [comment]
Maybe go to line 2588 and comment it.

**Comment by cyliamargatroid** [comment]
I did add an Game over sound and it worked. but no Continue music playing.

**Comment by Windblade-GR01** [comment]
It was introduced in 90c27b0e556e486183fa4b5d00bb0634a387506a
Fixed with commit bca6ac3afbf00e3a332d3932fe4894c10a420702
